x/concentrated-liquidity/position.go

```diff
@@ -62,7 +64,7 @@ func (k Keeper) initOrUpdatePosition(
 		return err
 	}
 
-	k.setPosition(ctx, poolId, owner, lowerTick, upperTick, joinTime, liquidity, positionId)
+	k.SetPosition(ctx, poolId, owner, lowerTick, upperTick, joinTime, liquidity, positionId, underlyingLockId)
```

Comment:
I'm guessing `noUnderlyingLockId` should be set here and `underlyingLockId` be removed from `initOrUpdatePosition` params?

Reply:
Yes, done here https://github.com/osmosis-labs/osmosis/pull/4882/commits/5855df51844dfb23cf09f734f05ca6d328afb1a1